Version 6.82.0 breaks AWS RIC support

## Expected Behavior

dd-lambda-js  works with container-based lambdas that are build using [AWS Lambda NodeJS Runtime Interface Client
](https://www.npmjs.com/package/aws-lambda-ric). AWS lambda RIC allows to create custom docker images to use them for lambda, which is useful when you need to create a lambda based on your own docker image.

## Actual Behavior

Since version `6.82.0` (`6.81.0` is fine), the file `node_modules/datadog-lambda-js/dist/handler.js` is not bundled anymore in the npm package. However AWS lambda ric doesn't seem to work without it, it throws an error `Cannot find module 'handler'`.
The code that throws is that one https://github.com/aws/aws-lambda-nodejs-runtime-interface-client/blob/main/src/utils/UserFunction.ts#L106

Because of this, all custom container based lambdas cannot be instrumented with version > 6.81.0.
So the problem is not really about the code itself, but the way it integrates with AWS RIC.

## Steps to Reproduce the Problem

1. Create a dummy lambda with AWS RIC => Follow the AWS RIC tuto (see doc in https://www.npmjs.com/package/aws-lambda-ric)
2. Add datadog docker layer + dd-lambda-js@6.82.0 => Follow the datadog tuto https://docs.datadoghq.com/serverless/installation/nodejs/?tab=containerimage
3. Build the container image
4. Run the image (I recommend adding AWS RIE cf https://docs.aws.amazon.com/gb_gb/lambda/latest/dg/images-test.html) (warning about this that will happen on version 6.81.0  https://github.com/aws/aws-lambda-runtime-interface-emulator/issues/65, for version 6.82.0 the problem doesn't happen since it throws before)
5. See error by calling running the lambda `curl --request POST "http://localhost:9000/2015-03-31/functions/function/invocations" -d '{}' --header 'Content-Type: application/json'` if you have done step 4

## Specifications

  - Datadog Lambda Layer version: 6.82.0
  - Node version: 16.16

## Stacktrace

From AWS lambda RIC:
  ```
{
  "errorType":"Runtime.ImportModuleError",
 "errorMessage":"Error: Cannot find module 'handler'\nRequire stack:\n- /function/node_modules/aws-lambda-ric/lib/utils/UserFunction.js\n
}
  ```

--- a/integration_tests/snapshots/logs/esm_node12.log
+++ b/integration_tests/snapshots/logs/esm_node12.log
@@ -1,18 +1,246 @@
 XXXX-XX-XX XX:XX:XX.XXX	ERROR	Uncaught Exception 	{"errorType":"Runtime.ImportModuleError","errorMessage":"Error: Cannot find module 'esm'\nRequire stack:\n- /opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js\n- /opt/nodejs/node_modules/datadog-lambda-js/runtime/index.js\n- /opt/nodejs/node_modules/datadog-lambda-js/handler.js\n- /var/runtime/UserFunction.js\n- /var/runtime/index.js","stack":["Runtime.ImportModuleError: Error: Cannot find module 'esm'","Require stack:","- /opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js","- /opt/nodejs/node_modules/datadog-lambda-js/runtime/index.js","- /opt/nodejs/node_modules/datadog-lambda-js/handler.js","- /var/runtime/UserFunction.js","- /var/runtime/index.js","    at ImportModuleError.ExtendedError [as constructor] (/opt/nodejs/node_modules/datadog-lambda-js/runtime/errors.js:XXX:XXX)","    at new ImportModuleError (/opt/nodejs/node_modules/datadog-lambda-js/runtime/errors.js:XXX:XXX)","    at _loadUserAppSync (/opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js:XXX:XXX)","    at loadSync (/opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js:XXX:XXX)","    at Object.<anonymous> (/opt/nodejs/node_modules/datadog-lambda-js/handler.js:XXX:XXX)","    at Module._compile (internal/modules/cjs/loader.js:XXX:XXX)","    at Object.Module._extensions..js (internal/modules/cjs/loader.js:XXX:XXX)","    at Module.load (internal/modules/cjs/loader.js:XXX:XXX)","    at Function.Module._load (internal/modules/cjs/loader.js:XXX:XXX)","    at Module.require (internal/modules/cjs/loader.js:XXX:XXX)"]}
+{
+  "traces": [
+    [
+      {
+        "trace_id":"XXXX",
+        "span_id":"XXXX",
+        "parent_id":"XXXX",
+        "name": "tcp.connect",
+        "resource": "127.0.0.1:9001",
+        "error": 0,
+        "meta": {
+          "_dd.p.dm": "-0",
+          "_dd.origin": "lambda",
+          "service": "integration-tests-js-XXXX-esm_node12",
+          "runtime-id":"XXXX",
+          "tcp.remote.host": "127.0.0.1",
+          "tcp.family": "IPv4",
+          "out.host": "127.0.0.1",
+          "span.kind": "client",
+          "tcp.local.address":"XXXX",
+          "language": "javascript"
+        },
+        "metrics": {
+          "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
+          "_dd.measured": 1,
+          "tcp.remote.port": 9001,
+          "out.port": 9001,
+          "tcp.local.port":XXXX,
+          "_sampling_priority_v1": 1
+        },
+        "start":XXXX,
+        "duration":XXXX,
+        "service": "integration-tests-js-XXXX-esm_node12"
+      }
+    ]
+  ]
+}
 START
 XXXX-XX-XX XX:XX:XX.XXX	ERROR	Uncaught Exception 	{"errorType":"Runtime.ImportModuleError","errorMessage":"Error: Cannot find module 'esm'\nRequire stack:\n- /opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js\n- /opt/nodejs/node_modules/datadog-lambda-js/runtime/index.js\n- /opt/nodejs/node_modules/datadog-lambda-js/handler.js\n- /var/runtime/UserFunction.js\n- /var/runtime/index.js","stack":["Runtime.ImportModuleError: Error: Cannot find module 'esm'","Require stack:","- /opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js","- /opt/nodejs/node_modules/datadog-lambda-js/runtime/index.js","- /opt/nodejs/node_modules/datadog-lambda-js/handler.js","- /var/runtime/UserFunction.js","- /var/runtime/index.js","    at ImportModuleError.ExtendedError [as constructor] (/opt/nodejs/node_modules/datadog-lambda-js/runtime/errors.js:XXX:XXX)","    at new ImportModuleError (/opt/nodejs/node_modules/datadog-lambda-js/runtime/errors.js:XXX:XXX)","    at _loadUserAppSync (/opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js:XXX:XXX)","    at loadSync (/opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js:XXX:XXX)","    at Object.<anonymous> (/opt/nodejs/node_modules/datadog-lambda-js/handler.js:XXX:XXX)","    at Module._compile (internal/modules/cjs/loader.js:XXX:XXX)","    at Object.Module._extensions..js (internal/modules/cjs/loader.js:XXX:XXX)","    at Module.load (internal/modules/cjs/loader.js:XXX:XXX)","    at Function.Module._load (internal/modules/cjs/loader.js:XXX:XXX)","    at Module.require (internal/modules/cjs/loader.js:XXX:XXX)"]}
+{
+  "traces": [
+    [
+      {
+        "trace_id":"XXXX",
+        "span_id":"XXXX",
+        "parent_id":"XXXX",
+        "name": "tcp.connect",
+        "resource": "127.0.0.1:9001",
+        "error": 0,
+        "meta": {
+          "_dd.p.dm": "-0",
+          "_dd.origin": "lambda",
+          "service": "integration-tests-js-XXXX-esm_node12",
+          "runtime-id":"XXXX",
+          "tcp.remote.host": "127.0.0.1",
+          "tcp.family": "IPv4",
+          "out.host": "127.0.0.1",
+          "span.kind": "client",
+          "tcp.local.address":"XXXX",
+          "language": "javascript"
+        },
+        "metrics": {
+          "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
+          "_dd.measured": 1,
+          "tcp.remote.port": 9001,
+          "out.port": 9001,
+          "tcp.local.port":XXXX,
+          "_sampling_priority_v1": 1
+        },
+        "start":XXXX,
+        "duration":XXXX,
+        "service": "integration-tests-js-XXXX-esm_node12"
+      }
+    ]
+  ]
+}
 END Duration: XXXX ms Memory Used: XXXX MB
 Unknown application error occurred
 Runtime.ImportModuleError
 START
 XXXX-XX-XX XX:XX:XX.XXX	ERROR	Uncaught Exception 	{"errorType":"Runtime.ImportModuleError","errorMessage":"Error: Cannot find module 'esm'\nRequire stack:\n- /opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js\n- /opt/nodejs/node_modules/datadog-lambda-js/runtime/index.js\n- /opt/nodejs/node_modules/datadog-lambda-js/handler.js\n- /var/runtime/UserFunction.js\n- /var/runtime/index.js","stack":["Runtime.ImportModuleError: Error: Cannot find module 'esm'","Require stack:","- /opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js","- /opt/nodejs/node_modules/datadog-lambda-js/runtime/index.js","- /opt/nodejs/node_modules/datadog-lambda-js/handler.js","- /var/runtime/UserFunction.js","- /var/runtime/index.js","    at ImportModuleError.ExtendedError [as constructor] (/opt/nodejs/node_modules/datadog-lambda-js/runtime/errors.js:XXX:XXX)","    at new ImportModuleError (/opt/nodejs/node_modules/datadog-lambda-js/runtime/errors.js:XXX:XXX)","    at _loadUserAppSync (/opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js:XXX:XXX)","    at loadSync (/opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js:XXX:XXX)","    at Object.<anonymous> (/opt/nodejs/node_modules/datadog-lambda-js/handler.js:XXX:XXX)","    at Module._compile (internal/modules/cjs/loader.js:XXX:XXX)","    at Object.Module._extensions..js (internal/modules/cjs/loader.js:XXX:XXX)","    at Module.load (internal/modules/cjs/loader.js:XXX:XXX)","    at Function.Module._load (internal/modules/cjs/loader.js:XXX:XXX)","    at Module.require (internal/modules/cjs/loader.js:XXX:XXX)"]}
+{
+  "traces": [
+    [
+      {
+        "trace_id":"XXXX",
+        "span_id":"XXXX",
+        "parent_id":"XXXX",
+        "name": "tcp.connect",
+        "resource": "127.0.0.1:9001",
+        "error": 0,
+        "meta": {
+          "_dd.p.dm": "-0",
+          "_dd.origin": "lambda",
+          "service": "integration-tests-js-XXXX-esm_node12",
+          "runtime-id":"XXXX",
+          "tcp.remote.host": "127.0.0.1",
+          "tcp.family": "IPv4",
+          "out.host": "127.0.0.1",
+          "span.kind": "client",
+          "tcp.local.address":"XXXX",
+          "language": "javascript"
+        },
+        "metrics": {
+          "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
+          "_dd.measured": 1,
+          "tcp.remote.port": 9001,
+          "out.port": 9001,
+          "tcp.local.port":XXXX,
+          "_sampling_priority_v1": 1
+        },
+        "start":XXXX,
+        "duration":XXXX,
+        "service": "integration-tests-js-XXXX-esm_node12"
+      }
+    ]
+  ]
+}
 END Duration: XXXX ms Memory Used: XXXX MB
 Unknown application error occurred
 Runtime.ImportModuleError
 XXXX-XX-XX XX:XX:XX.XXX	ERROR	Uncaught Exception 	{"errorType":"Runtime.ImportModuleError","errorMessage":"Error: Cannot find module 'esm'\nRequire stack:\n- /opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js\n- /opt/nodejs/node_modules/datadog-lambda-js/runtime/index.js\n- /opt/nodejs/node_modules/datadog-lambda-js/handler.js\n- /var/runtime/UserFunction.js\n- /var/runtime/index.js","stack":["Runtime.ImportModuleError: Error: Cannot find module 'esm'","Require stack:","- /opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js","- /opt/nodejs/node_modules/datadog-lambda-js/runtime/index.js","- /opt/nodejs/node_modules/datadog-lambda-js/handler.js","- /var/runtime/UserFunction.js","- /var/runtime/index.js","    at ImportModuleError.ExtendedError [as constructor] (/opt/nodejs/node_modules/datadog-lambda-js/runtime/errors.js:XXX:XXX)","    at new ImportModuleError (/opt/nodejs/node_modules/datadog-lambda-js/runtime/errors.js:XXX:XXX)","    at _loadUserAppSync (/opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js:XXX:XXX)","    at loadSync (/opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js:XXX:XXX)","    at Object.<anonymous> (/opt/nodejs/node_modules/datadog-lambda-js/handler.js:XXX:XXX)","    at Module._compile (internal/modules/cjs/loader.js:XXX:XXX)","    at Object.Module._extensions..js (internal/modules/cjs/loader.js:XXX:XXX)","    at Module.load (internal/modules/cjs/loader.js:XXX:XXX)","    at Function.Module._load (internal/modules/cjs/loader.js:XXX:XXX)","    at Module.require (internal/modules/cjs/loader.js:XXX:XXX)"]}
+{
+  "traces": [
+    [
+      {
+        "trace_id":"XXXX",
+        "span_id":"XXXX",
+        "parent_id":"XXXX",
+        "name": "tcp.connect",
+        "resource": "127.0.0.1:9001",
+        "error": 0,
+        "meta": {
+          "_dd.p.dm": "-0",
+          "_dd.origin": "lambda",
+          "service": "integration-tests-js-XXXX-esm_node12",
+          "runtime-id":"XXXX",
+          "tcp.remote.host": "127.0.0.1",
+          "tcp.family": "IPv4",
+          "out.host": "127.0.0.1",
+          "span.kind": "client",
+          "tcp.local.address":"XXXX",
+          "language": "javascript"
+        },
+        "metrics": {
+          "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
+          "_dd.measured": 1,
+          "tcp.remote.port": 9001,
+          "out.port": 9001,
+          "tcp.local.port":XXXX,
+          "_sampling_priority_v1": 1
+        },
+        "start":XXXX,
+        "duration":XXXX,
+        "service": "integration-tests-js-XXXX-esm_node12"
+      }
+    ]
+  ]
+}
 START
 XXXX-XX-XX XX:XX:XX.XXX	ERROR	Uncaught Exception 	{"errorType":"Runtime.ImportModuleError","errorMessage":"Error: Cannot find module 'esm'\nRequire stack:\n- /opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js\n- /opt/nodejs/node_modules/datadog-lambda-js/runtime/index.js\n- /opt/nodejs/node_modules/datadog-lambda-js/handler.js\n- /var/runtime/UserFunction.js\n- /var/runtime/index.js","stack":["Runtime.ImportModuleError: Error: Cannot find module 'esm'","Require stack:","- /opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js","- /opt/nodejs/node_modules/datadog-lambda-js/runtime/index.js","- /opt/nodejs/node_modules/datadog-lambda-js/handler.js","- /var/runtime/UserFunction.js","- /var/runtime/index.js","    at ImportModuleError.ExtendedError [as constructor] (/opt/nodejs/node_modules/datadog-lambda-js/runtime/errors.js:XXX:XXX)","    at new ImportModuleError (/opt/nodejs/node_modules/datadog-lambda-js/runtime/errors.js:XXX:XXX)","    at _loadUserAppSync (/opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js:XXX:XXX)","    at loadSync (/opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js:XXX:XXX)","    at Object.<anonymous> (/opt/nodejs/node_modules/datadog-lambda-js/handler.js:XXX:XXX)","    at Module._compile (internal/modules/cjs/loader.js:XXX:XXX)","    at Object.Module._extensions..js (internal/modules/cjs/loader.js:XXX:XXX)","    at Module.load (internal/modules/cjs/loader.js:XXX:XXX)","    at Function.Module._load (internal/modules/cjs/loader.js:XXX:XXX)","    at Module.require (internal/modules/cjs/loader.js:XXX:XXX)"]}
+{
+  "traces": [
+    [
+      {
+        "trace_id":"XXXX",
+        "span_id":"XXXX",
+        "parent_id":"XXXX",
+        "name": "tcp.connect",
+        "resource": "127.0.0.1:9001",
+        "error": 0,
+        "meta": {
+          "_dd.p.dm": "-0",
+          "_dd.origin": "lambda",
+          "service": "integration-tests-js-XXXX-esm_node12",
+          "runtime-id":"XXXX",
+          "tcp.remote.host": "127.0.0.1",
+          "tcp.family": "IPv4",
+          "out.host": "127.0.0.1",
+          "span.kind": "client",
+          "tcp.local.address":"XXXX",
+          "language": "javascript"
+        },
+        "metrics": {
+          "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
+          "_dd.measured": 1,
+          "tcp.remote.port": 9001,
+          "out.port": 9001,
+          "tcp.local.port":XXXX,
+          "_sampling_priority_v1": 1
+        },
+        "start":XXXX,
+        "duration":XXXX,
+        "service": "integration-tests-js-XXXX-esm_node12"
+      }
+    ]
+  ]
+}
 END Duration: XXXX ms Memory Used: XXXX MB
 Unknown application error occurred
 Runtime.ImportModuleError
 XXXX-XX-XX XX:XX:XX.XXX	ERROR	Uncaught Exception 	{"errorType":"Runtime.ImportModuleError","errorMessage":"Error: Cannot find module 'esm'\nRequire stack:\n- /opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js\n- /opt/nodejs/node_modules/datadog-lambda-js/runtime/index.js\n- /opt/nodejs/node_modules/datadog-lambda-js/handler.js\n- /var/runtime/UserFunction.js\n- /var/runtime/index.js","stack":["Runtime.ImportModuleError: Error: Cannot find module 'esm'","Require stack:","- /opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js","- /opt/nodejs/node_modules/datadog-lambda-js/runtime/index.js","- /opt/nodejs/node_modules/datadog-lambda-js/handler.js","- /var/runtime/UserFunction.js","- /var/runtime/index.js","    at ImportModuleError.ExtendedError [as constructor] (/opt/nodejs/node_modules/datadog-lambda-js/runtime/errors.js:XXX:XXX)","    at new ImportModuleError (/opt/nodejs/node_modules/datadog-lambda-js/runtime/errors.js:XXX:XXX)","    at _loadUserAppSync (/opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js:XXX:XXX)","    at loadSync (/opt/nodejs/node_modules/datadog-lambda-js/runtime/user-function.js:XXX:XXX)","    at Object.<anonymous> (/opt/nodejs/node_modules/datadog-lambda-js/handler.js:XXX:XXX)","    at Module._compile (internal/modules/cjs/loader.js:XXX:XXX)","    at Object.Module._extensions..js (internal/modules/cjs/loader.js:XXX:XXX)","    at Module.load (internal/modules/cjs/loader.js:XXX:XXX)","    at Function.Module._load (internal/modules/cjs/loader.js:XXX:XXX)","    at Module.require (internal/modules/cjs/loader.js:XXX:XXX)"]}
+{
+  "traces": [
+    [
+      {
+        "trace_id":"XXXX",
+        "span_id":"XXXX",
+        "parent_id":"XXXX",
+        "name": "tcp.connect",
+        "resource": "127.0.0.1:9001",
+        "error": 0,
+        "meta": {
+          "_dd.p.dm": "-0",
+          "_dd.origin": "lambda",
+          "service": "integration-tests-js-XXXX-esm_node12",
+          "runtime-id":"XXXX",
+          "tcp.remote.host": "127.0.0.1",
+          "tcp.family": "IPv4",
+          "out.host": "127.0.0.1",
+          "span.kind": "client",
+          "tcp.local.address":"XXXX",
+          "language": "javascript"
+        },
+        "metrics": {
+          "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
+          "_dd.measured": 1,
+          "tcp.remote.port": 9001,
+          "out.port": 9001,
+          "tcp.local.port":XXXX,
+          "_sampling_priority_v1": 1
+        },
+        "start":XXXX,
+        "duration":XXXX,
+        "service": "integration-tests-js-XXXX-esm_node12"
+      }
+    ]
+  ]
+}

--- a/integration_tests/snapshots/logs/process-input-traced_node12.log
+++ b/integration_tests/snapshots/logs/process-input-traced_node12.log
@@ -34,6 +34,7 @@ START
         "resource": "GET /{proxy+}",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "runtime-id":"XXXX",
           "operation_name": "aws.apigateway",
           "resource_names": "GET /{proxy+}",
@@ -47,6 +48,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "_sampling_priority_v1": 1
         },
         "start":XXXX,
@@ -168,6 +170,7 @@ START
         "resource": "sns-lambda",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "service": "sns",
           "runtime-id":"XXXX",
           "operation_name": "aws.sns",
@@ -185,6 +188,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "_sampling_priority_v1": 1
         },
         "start":XXXX,
@@ -284,6 +288,7 @@ START
         "resource": "my-queue",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "service": "sqs",
           "runtime-id":"XXXX",
           "operation_name": "aws.sqs",
@@ -298,6 +303,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "retry_count": 1,
           "_sampling_priority_v1": 1
         },

--- a/integration_tests/snapshots/logs/process-input-traced_node14.log
+++ b/integration_tests/snapshots/logs/process-input-traced_node14.log
@@ -34,6 +34,7 @@ START
         "resource": "GET /{proxy+}",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "runtime-id":"XXXX",
           "operation_name": "aws.apigateway",
           "resource_names": "GET /{proxy+}",
@@ -47,6 +48,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "_sampling_priority_v1": 1
         },
         "start":XXXX,
@@ -168,6 +170,7 @@ START
         "resource": "sns-lambda",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "service": "sns",
           "runtime-id":"XXXX",
           "operation_name": "aws.sns",
@@ -185,6 +188,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "_sampling_priority_v1": 1
         },
         "start":XXXX,
@@ -284,6 +288,7 @@ START
         "resource": "my-queue",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "service": "sqs",
           "runtime-id":"XXXX",
           "operation_name": "aws.sqs",
@@ -298,6 +303,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "retry_count": 1,
           "_sampling_priority_v1": 1
         },

--- a/integration_tests/snapshots/logs/process-input-traced_node16.log
+++ b/integration_tests/snapshots/logs/process-input-traced_node16.log
@@ -34,6 +34,7 @@ START
         "resource": "GET /{proxy+}",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "version": "1.0.0",
           "runtime-id":"XXXX",
           "operation_name": "aws.apigateway",
@@ -48,6 +49,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "_sampling_priority_v1": 1
         },
         "start":XXXX,
@@ -172,6 +174,7 @@ START
         "resource": "sns-lambda",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "service": "sns",
           "version": "1.0.0",
           "runtime-id":"XXXX",
@@ -190,6 +193,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "_sampling_priority_v1": 1
         },
         "start":XXXX,
@@ -291,6 +295,7 @@ START
         "resource": "my-queue",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "service": "sqs",
           "version": "1.0.0",
           "runtime-id":"XXXX",
@@ -306,6 +311,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "retry_count": 1,
           "_sampling_priority_v1": 1
         },

--- a/integration_tests/snapshots/logs/status-code-500s_node12.log
+++ b/integration_tests/snapshots/logs/status-code-500s_node12.log
@@ -40,6 +40,7 @@ START
         "resource": "GET /{proxy+}",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "runtime-id":"XXXX",
           "operation_name": "aws.apigateway",
           "resource_names": "GET /{proxy+}",
@@ -53,6 +54,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "_sampling_priority_v1": 1
         },
         "start":XXXX,
@@ -121,6 +123,7 @@ START
         "resource": "sns-lambda",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "service": "sns",
           "runtime-id":"XXXX",
           "operation_name": "aws.sns",
@@ -138,6 +141,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "_sampling_priority_v1": 1
         },
         "start":XXXX,
@@ -205,6 +209,7 @@ START
         "resource": "my-queue",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "service": "sqs",
           "runtime-id":"XXXX",
           "operation_name": "aws.sqs",
@@ -219,6 +224,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "retry_count": 1,
           "_sampling_priority_v1": 1
         },

--- a/integration_tests/snapshots/logs/status-code-500s_node14.log
+++ b/integration_tests/snapshots/logs/status-code-500s_node14.log
@@ -40,6 +40,7 @@ START
         "resource": "GET /{proxy+}",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "runtime-id":"XXXX",
           "operation_name": "aws.apigateway",
           "resource_names": "GET /{proxy+}",
@@ -53,6 +54,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "_sampling_priority_v1": 1
         },
         "start":XXXX,
@@ -121,6 +123,7 @@ START
         "resource": "sns-lambda",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "service": "sns",
           "runtime-id":"XXXX",
           "operation_name": "aws.sns",
@@ -138,6 +141,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "_sampling_priority_v1": 1
         },
         "start":XXXX,
@@ -205,6 +209,7 @@ START
         "resource": "my-queue",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "service": "sqs",
           "runtime-id":"XXXX",
           "operation_name": "aws.sqs",
@@ -219,6 +224,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "retry_count": 1,
           "_sampling_priority_v1": 1
         },

--- a/integration_tests/snapshots/logs/status-code-500s_node16.log
+++ b/integration_tests/snapshots/logs/status-code-500s_node16.log
@@ -40,6 +40,7 @@ START
         "resource": "GET /{proxy+}",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "version": "1.0.0",
           "runtime-id":"XXXX",
           "operation_name": "aws.apigateway",
@@ -54,6 +55,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "_sampling_priority_v1": 1
         },
         "start":XXXX,
@@ -123,6 +125,7 @@ START
         "resource": "sns-lambda",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "service": "sns",
           "version": "1.0.0",
           "runtime-id":"XXXX",
@@ -141,6 +144,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "_sampling_priority_v1": 1
         },
         "start":XXXX,
@@ -209,6 +213,7 @@ START
         "resource": "my-queue",
         "error": 0,
         "meta": {
+          "_dd.p.dm": "-0",
           "service": "sqs",
           "version": "1.0.0",
           "runtime-id":"XXXX",
@@ -224,6 +229,7 @@ START
         },
         "metrics": {
           "_dd.agent_psr": 1,
+          "_dd.top_level": 1,
           "retry_count": 1,
           "_sampling_priority_v1": 1
         },

--- a/integration_tests/snapshots/logs/throw-error-traced_node12.log
+++ b/integration_tests/snapshots/logs/throw-error-traced_node12.log
@@ -65,6 +65,7 @@ START
 XXXX-XX-XX XX:XX:XX.XXX	ERROR	[dd.trace_id=XXXX dd.span_id=XXXX] Invoke Error 	{"errorType":"Error","errorMessage":"Hello","stack":["Error: Hello","    at handle (/var/task/throw-error-traced.js:XXX:XXX)","    at /opt/nodejs/node_modules/datadog-lambda-js/utils/handler.js:XXX:XXX","    at /opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX","    at step (/opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX)","    at Object.next (/opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX)","    at /opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX","    at new Promise (<anonymous>)","    at __awaiter (/opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX)","    at /opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX","    at /opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX"]}
 END Duration: XXXX ms Memory Used: XXXX MB
 START
+XXXX-XX-XX XX:XX:XX.XXX	ERROR	[dd.trace_id=XXXX dd.span_id=XXXX] Invoke Error 	{"errorType":"Error","errorMessage":"Hello","stack":["Error: Hello","    at handle (/var/task/throw-error-traced.js:XXX:XXX)","    at /opt/nodejs/node_modules/datadog-lambda-js/utils/handler.js:XXX:XXX","    at /opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX","    at step (/opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX)","    at Object.next (/opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX)","    at /opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX","    at new Promise (<anonymous>)","    at __awaiter (/opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX)","    at /opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX","    at /opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX"]}
 {
   "e": XXXX,
   "m": "aws.lambda.enhanced.invocations",
@@ -95,5 +96,4 @@ START
   ],
   "v": 1
 }
-XXXX-XX-XX XX:XX:XX.XXX	ERROR	[dd.trace_id=XXXX dd.span_id=XXXX] Invoke Error 	{"errorType":"Error","errorMessage":"Hello","stack":["Error: Hello","    at handle (/var/task/throw-error-traced.js:XXX:XXX)","    at /opt/nodejs/node_modules/datadog-lambda-js/utils/handler.js:XXX:XXX","    at /opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX","    at step (/opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX)","    at Object.next (/opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX)","    at /opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX","    at new Promise (<anonymous>)","    at __awaiter (/opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX)","    at /opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX","    at /opt/nodejs/node_modules/datadog-lambda-js/index.js:XXX:XXX"]}
 END Duration: XXXX ms Memory Used: XXXX MB

--- a/scripts/publish_prod.sh
+++ b/scripts/publish_prod.sh
@@ -76,7 +76,11 @@ if [ "$CONT" != "y" ]; then
 fi
 
 echo 'Publishing to NPM'
+if [ -d "./dist" ]; then
+    rm -rf ./dist
+fi
 yarn build
+cp ./dist/handler.cjs ./dist/handler.js
 yarn publish --new-version "$NEW_VERSION"
 
 echo

